packages/tupaia-web/src/features/Map/utils/useMapOverlayMapData.ts

```diff
@@ -13,6 +13,30 @@ import {
 import { Entity } from '../../../types';
 import { useMapOverlayTableData } from './useMapOverlayTableData';
 
+// if the entity is a point, filter it out so that we don't end up with navigation points showing no data type. This ensures that the only points on the map are from an overlay (see: [RN-1328](https://linear.app/bes/issue/RN-1328/entities-can-appear-as-black-smudges-on-map) for more context)
```

Comment:
```suggestion
/**
 * If the entity is a point, filter it out so that we don’t end up with 
 * navigation points showing no data type. This ensures that the only points on 
 * the map are from an overlay.
 *
 * @see https://linear.app/bes/issue/RN-1328/entities-can-appear-as-black-smudges-on-map
 */
```